user_docs/en/changes.t2t

```diff
@@ -926,7 +926,7 @@ Highlights of this release include Support for tables in Kindle for PC, support
 
 
 = 2018.1.1 =
-This is a special release of NVDA which addresses   a bug in the Onecore Windows Speech synthesizer driver, which was causing it to speak with a higher pitch and speed in Windows 10 Redstone 4 (1803). (#8082)  
+This is a special release of NVDA which addresses  a bug in the Onecore Windows Speech synthesizer driver, which was causing it to speak with a higher pitch and speed in Windows 10 Redstone 4 (1803). (#8082)  

```

Comment:
This removed a space, but there is still a double space here. IE it was a triple space.